src/main/java/core/constants/player/Speciality.java

```diff
@@ -1,7 +1,7 @@
 package core.constants.player;
 	// enum for speciality
 	public enum Speciality {
-		NO_SPECIALITY(0), TECHNICAL(1), QUICK(2), POWERFUL(3), UNPREDICTABLE(4), HEAD(5), REGAINER(6), SUPPORT(8);
+		NO_SPECIALITY(0), TECHNICAL(1), QUICK(2), POWERFUL(3), UNPREDICTABLE(4), HEAD(5), REGAINER(6), _UNUSED(7), SUPPORT(8);
```

Comment:
it might have impact on other places ... 
this one will need to be tested or at least we need to check where that enum is being used using ordinal value ...

Reply:
this is what i meant- there are several instructions like this one
        Speciality s = Speciality.values()[iPlayerSpecialty];

they all will fail if iPlayerSpecialty is SUPPORTER

i don't find any usage, where _UNUSED would make trouble